kentikapi/client_integration_test.go

```diff
@@ -212,6 +211,8 @@ func TestClient_GetAgent(t *testing.T) {
 				proto.Equal(tt.expectedResult, result),
 				fmt.Sprintf("Protobuf messages are not equal:\nexpected: %v\nactual:  %v", tt.expectedResult, result),
 			)
+			s.Stop()
+			assert.True(t, <-s.done)
```

Comment:
I think that it should be moved to `(s *spySyntheticsServer) Stop() {}` function, so that the the user has only to call one method to stop the server.

Reply:
Good idea